packages/node_modules/@webex/plugin-meetings/test/integration/spec/journey.js

```diff
@@ -706,6 +706,89 @@ skipInNode(describe)('plugin-meetings', () => {
           throw e;
         }));
 
+      it('alice resolution is 1080p, check what bob receives', (done) => {
+        testUtils.delayedPromise(alice.meeting.setLocalVideoQuality('1080p'))
+          .then(() => {
+            testUtils.waitUntil(2000);
```

Comment:
what's the purpose of this call here? was it supposed to add a delay before the checks are done? It doesn't do that, because the return value is ignored

Reply:
My bad, removed. Used interval wrapped in a promise instead.